foursight_core/checks/ecs_recovery_check.py

```diff
@@ -63,6 +73,6 @@ def rollback_application_version(connection, **kwargs):
                 return check
         else:
             check.status = 'PASS'
-            check.summary = 'No active deployment detected - all good!'
+            check.summary = 'No active deployment detected (or counts are uneven) - all good!'
```

Comment:
I don't really like the hyphen here. I'd rather see you put a period and start a new sentence. But the more compelling problem is that I just don't understand what's being said. Does this really mean "Either there are no active deployments or else the counts are uneven" ?? That seems a really odd set of things to put together in an `or`.